src/command/render/latexmk/texlive.ts

```diff
@@ -53,7 +54,7 @@ export async function findPackages(
     } else {
       const result = await tlmgrCommand(
         "search",
-        [...args, ...(opts || []), searchTerm],
+        [...args, ...(opts || []), `"${searchTerm}"`],
```

Comment:
I think those quotes were part of the problem also and need to be added for CMD to work

Reply:
We quote every argument now for windows